lib/chef/knife/core/gem_glob_loader.rb

```diff
@@ -22,7 +22,7 @@ class Chef
   class Knife
     class SubcommandLoader
       class GemGlobLoader < Chef::Knife::SubcommandLoader
-        MATCHES_CHEF_GEM ||= %r{/chef-[\d]+\.[\d]+\.[\d]+}.freeze
+        MATCHES_CHEF_GEM ||= %r{/che-[\d+.[\d+.[\d+}.freeze
```

Comment:
```suggestion
        MATCHES_CHEF_GEM ||= %r{/chef-[\d+.[\d+.[\d+}.freeze
```

Reply:
This one got pretty mangled, was this autocorrect or manual?

Reply:
Issue filed here: https://github.com/rubocop-hq/rubocop/issues/8948